tests/test_slicker.py

```diff
@@ -174,6 +174,8 @@ def test_moving_implicit(self):
             'moving_implicit',
             'foo.secrets.lulz', 'quux.new_name')
 
+    @unittest.skip("TO DISCUSS fix_python_imports is now moved below "
```

Comment:
Heh, this one it's probably easiest to just change the import to be `from third_party import fix_python_imports`, in which case the `.isort.cfg` I mention will suffice.

Reply:
I'll give it a try

Reply:
When I try with the following:
```
[settings]
known_third_party=fix_python_imports
```
It is different:

```diff
+ import fix_python_imports
+
  import codemod_fork as the_other_codemod
```

Reply:
Oh, I see.  This is a real issue with `isort` IMO -- see timothycrosley/isort#468 -- but we can just work around it for now -- simplest is probably to just remove the `sys.path` line from the test data, and resort both input and output to do what `isort` wants.

Though this makes me wonder: until that issue is fixed, is it better to simply skip isort for any file that has code before imports?  That would be a little sad -- it's a slight regression from fix-includes I think -- but maybe better than making a mess of it.  Maybe @csilvers has an opinion on this question.

Reply:
I definitely think we don't want to let isort sort imports around code -- that will break things, like in the example at https://github.com/timothycrosley/isort/issues/693.  Trying to work around the problem is easy for tests but not for actual real code.

I think what we'd have to do is to change slicker to notice blocks of imports without any other code, call isort on those blocks individually, and then merge them all together.  I don't even know if isort will work that way, or if it needs to know the entire file contents.

It may just be easiest to hold off on porting to isort until this is fixed.  Sad though.